README.md

```diff
@@ -43,35 +44,38 @@ SNP_id
 </pre>
 </details>
 
-<details>
-<summary>Example of table with -t:</summary>
-<pre>
-head tests/variants_query_table.txt
-chr_id,position,ref_allele,alt_allele
-1,154453788,C,T
-1,1022868,A,G
-1,2211079,A,C
-1,2293397,G,A
-1,6568959,A,AG
-1,8094061,TG,T
-1,8447713,G,A
-1,9283562,C,T
-1,9478595,G,C
-</pre>
-</details>
+## Type of annotation
 
-To run the program from a srun you can use also the script v2g_dask_query.py after loading the tiledb conda environment:
+Two  type of annotation are given:
+    -   The variant to gene which will return all the scores that associate a variant with a gene for all the QTL types.
+    -   The variant to disease which return all the annotation that associate a variant with a trait from a GWAS.
 
-```
-conda activate tiledb
-python v2g_dask_query.py -t <variant-table> -o <output-file>
-```
+## Optional parameters
+
+In the variant_disease_gene option the following optional parameters are available:
+
+    --tag which will match the query variants with the tag variants from OTG instead than using the lead ones (Default: false).
+
+    --gnomad_af which will return the allele frequency for the alternate allele from a specific population (Default: gnomad_nfe). The list of population availablefor this parameter are reported below
+
+### Populations for the --gnomad_af parameter
+    -   gnomad_nfe non-Finnish Europeans
+    -   gnomad_afr Africans
+    -   gnomad_amr North Americans
+    -   gnomad_asj Ashkenazi jew
+    -   gnomad_eas East Asians
+    -   gnomad_fin Finnish
+    -   gnomad_nfe_est non-Finnish East Europeans
+    -   gnomad_nfe_nwe non-Finnish North-Western Europeans
+    -   gnomad_nfe_seu non-Finnish South-Eastern Europeans
+    -   gnomad_nfe_onf other non-Finnish Europeans
+    -   gnomad_oth other
 
 ## Output
 
+### variant_gene output
 The output from the program look like the one described below when running the sbatch commands described above:
 
```

Comment:
Add here a short explanation of the output fields, especially describe the list values.

Reply:
A description is given below in the output section